.travis.yml

```diff
@@ -2,10 +2,12 @@ dist: xenial
 sudo: false
 
 rvm:
-  - "2.0"
   - "2.1"
   - "2.2"
   - "2.3"
+  - "2.4"
```

Comment:
No blocker, but given we only actually support Omnibus-packaged Aptible Toolbelt binaries, should we trim this matrix to just the version(s) of Ruby we use in omnibus-aptible-toolbelt?

Reply:
Yes, but I don't think we have that documented. I know there's a different Ruby version for OSX vs Ubuntu, and I never looked at which Windows uses.  One of the many areas of tech debt with the CLI.